From e94db263f63e17a80636f9062c6f1e0f66c262e3 Mon Sep 17 00:00:00 2001
From: Tomasz Maciej Nowak <tomek_n@o2.pl>
Date: Wed, 24 Jan 2018 16:39:48 +0100
Subject: [PATCH 6/9] mvebu: rename clearfog sysupgrade script and functions

Cosmetic name change of ClearFog sysupgrade script and its functions to
more generic sdcard. This way it won't be confusing for other future
device additions.

Signed-off-by: Tomasz Maciej Nowak <tomek_n@o2.pl>
---
 target/linux/mvebu/base-files/lib/upgrade/platform.sh               | 4 ++--
 .../linux/mvebu/base-files/lib/upgrade/{clearfog.sh => sdcard.sh}   | 6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)
 rename target/linux/mvebu/base-files/lib/upgrade/{clearfog.sh => sdcard.sh} (83%)

diff --git a/target/linux/mvebu/base-files/lib/upgrade/platform.sh b/target/linux/mvebu/base-files/lib/upgrade/platform.sh
index 8241d1edcf..9334251246 100755
--- a/target/linux/mvebu/base-files/lib/upgrade/platform.sh
+++ b/target/linux/mvebu/base-files/lib/upgrade/platform.sh
@@ -17,7 +17,7 @@ platform_do_upgrade() {
 		platform_do_upgrade_linksys "$ARGV"
 		;;
 	armada-388-clearfog-base|armada-388-clearfog-pro)
-		platform_do_upgrade_clearfog "$ARGV"
+		platform_do_upgrade_sdcard "$ARGV"
 		;;
 	*)
 		default_do_upgrade "$ARGV"
@@ -30,7 +30,7 @@ platform_copy_config() {
 		platform_copy_config_linksys
 		;;
 	armada-388-clearfog-base|armada-388-clearfog-pro)
-		platform_copy_config_clearfog "$ARGV"
+		platform_copy_config_sdcard "$ARGV"
 		;;
 	esac
 }
diff --git a/target/linux/mvebu/base-files/lib/upgrade/clearfog.sh b/target/linux/mvebu/base-files/lib/upgrade/sdcard.sh
similarity index 83%
rename from target/linux/mvebu/base-files/lib/upgrade/clearfog.sh
rename to target/linux/mvebu/base-files/lib/upgrade/sdcard.sh
index b6dbde8458..de6e1a920c 100644
--- a/target/linux/mvebu/base-files/lib/upgrade/clearfog.sh
+++ b/target/linux/mvebu/base-files/lib/upgrade/sdcard.sh
@@ -4,7 +4,7 @@ get_magic_at() {
 	get_image "$file" | dd bs=1 count=2 skip="$pos" 2>/dev/null | hexdump -v -n 2 -e '1/1 "%02x"'
 }
 
-platform_check_image_clearfog() {
+platform_check_image_sdcard() {
 	local file="$1"
 	local magic
 
@@ -17,13 +17,13 @@ platform_check_image_clearfog() {
 	return 0;
 }
 
-platform_do_upgrade_clearfog() {
+platform_do_upgrade_sdcard() {
 	sync
 	get_image "$1" | dd of=/dev/mmcblk0 bs=2M conv=fsync
 	sleep 1
 }
 
-platform_copy_config_clearfog() {
+platform_copy_config_sdcard() {
 	mkdir -p /boot
 	[ -f /boot/kernel.img ] || mount -o rw,noatime /dev/mmcblk0p1 /boot
 	cp -af "$CONF_TAR" /boot/
-- 
2.16.2

